views/scan.hbs

```diff
@@ -8,9 +8,21 @@
       {{#if specificBreach}}
         {{#getScanResultsHeadline }}
           <h2 class="headline scan-results-headline fb-res-headline">{{{ this.headline }}}</h2>
-          {{#if this.subhead }}
-            <h3 class="scan-res-subhead">{{{ this.subhead }}}</h2>
-          {{/if}}
+            {{#ifCompare ../specificBreach.Domain "===" "facebook.com"}}
+            <div class="facebook-notes">
+            <p>Your email doesn’t appear in this breach, but your phone number may still be vulnerable. Some of the accounts compromised in the Facebook breach included phone numbers and other personal information but not email addresses. If you have ever signed up for a Facebook account -- even if you don’t use it now -- we recommend you take these three simple steps to protect yourself:
+</p>
+            <h3>What to do</h3>
+            <ol>
+            <li><strong>Change the passcode on your mobile phone carrier accounts to prevent SIM-jacking.</strong> SIM-jacking is when a hacker uses phone numbers, date of birth and other data to take over a person’s cell phone number and then hack into their email, social media and even financial accounts.</li>
+            <li><strong>Change your profile information to private in your Facebook privacy settings.</strong> During this breach, hackers took profile information that was set as “open to the public” or “shared with friends.” This information can be combined with other data to access even more of your personal information and accounts.</li>
+            <li>Follow the recommendations on <a href="/breach-details/Facebook">our Facebook breach page</a>.</li>
+            </div>
+            {{else}}
+              {{#if this.subhead }}
+                <h3 class="scan-res-subhead">{{{ this.subhead }}}</h2>
```

Comment:
Something is off here. We open an `<h3>` but close an `</h2>`. Not sure if it should be an H3 or H2 (but I'm guessing an H3 if we're nested under the `H2` headline).

Reply:
Fixed.